internal/operators/openshiftai/openshift_ai_operator_test.go

```diff
@@ -171,5 +173,111 @@ var _ = Describe("Operator", func() {
 			},
 			true,
 		),
+		Entry(
+			"NVIDIA is supported even if not explicitly added",
+			map[string]string{
+				"OPENSHIFT_AI_SUPPORTED_GPUS": "1af4",
```

Comment:
Trying to understand why 1AF4 is here?

Looking up the vendor ID I can see that this is a Virtio device? (of which GPU can be one?)

https://devicehunt.com/search/type/pci/vendor/1AF4/device/any

Reply:
Also, make this a constant, same as the suggestion for the Nvidia one

Reply:
Do we need a test case for the scenario where the Nvidia GPU has not been explicitly added?

Reply:
As you discovered `14fa` is the vendor identifier of VirtIO devices. We use it only for testing purposes. Setting the `OPENSHIFT_AI_SUPPORTED_GPUS` environment variable to `14fa` developers and QE engineers can test the OpenShift AI operator feature, specially the validations, without requiring an actual NVIDIA GPU.

I will add the constant and the test case.

Reply:
Actually there is already a test case for the scenario where NVIDIA is not explicitly added to `OPENSHIFT_AI_SUPPORTED_GPUS`, look for `NVIDIA is supported even if not explicitly added` in this test.